frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java

```diff
@@ -1312,7 +1312,7 @@ private void testLoggingUnload(Channel inferChannel, Channel mgmtChannel)
         Scanner logscanner = new Scanner(logfile, "UTF-8");
         while (logscanner.hasNextLine()) {
             String line = logscanner.nextLine();
-            if (line.contains("LoggingService exit")) {
+            if (line.contains("Model logging unregistered")) {
```

Comment:
Whats this change? I don't see corresponding change in the source file. Curious to know how this log line gets into the logfile.